app/over_react_redux/todo_client/lib/src/components/task_count.dart

```diff
@@ -9,17 +9,52 @@ import 'package:react_material_ui/react_material_ui.dart';
 
 part 'task_count.over_react.g.dart';
 
-UiFactory<TaskCountBadgeProps> TaskCountBadge = connect<AppState, TaskCountBadgeProps>(
-    mapStateToPropsWithOwnProps: (state, ownProps) {
-      return (TaskCountBadge()
-        ..assignedTodoIds = state.todos.where((todo) => todo.assignedUserId == ownProps.user.id)
-            .map((todo) => todo.id).toList()
-      );
-    },
-    areStatePropsEqual: (nextProps, prevProps) {
-      return ListEquality().equals(nextProps.assignedTodoIds, prevProps.assignedTodoIds);
-    },
-)(_$TaskCountBadge); // ignore: undefined_identifier
+UiFactory<TaskCountBadgeProps> TaskCountBadge =
+    connect<AppState, TaskCountBadgeProps>(
+  mapStateToPropsWithOwnProps: (state, ownProps) {
+    return (TaskCountBadge()
+      ..assignedTodoIds = state.todos
+          .where((todo) => todo.assignedUserId == ownProps.user.id)
+          .map((todo) => todo.id)
+          .toList());
+  },
+  areStatePropsEqual: (nextProps, prevProps) {
+    return ListEquality()
+        .equals(nextProps.assignedTodoIds, prevProps.assignedTodoIds);
+  },
+)(uiFunction((props) {
+  String _getTooltipContent() {
+    final taskCount = props.assignedTodoIds.length;
+    if (taskCount == 1) {
+      return 'There is $taskCount task assigned to ${props.user.name}';
+    }
+
+    return 'There are $taskCount tasks assigned to ${props.user.name}';
+  }
+
+  return (Box()
+    ..onMouseEnter = (_) {
+      props.dispatch(HighlightTodosAction(props.assignedTodoIds));
+    }
+    ..onMouseLeave = (_) {
+      props.dispatch(UnHighlightTodosAction(props.assignedTodoIds));
+    })(
+    (Tooltip()
+      ..title = _getTooltipContent()
+      ..arrow = true
+      ..enterDelay = 500)(
+      (Badge()
+        ..badgeContent = Dom.span()(props.assignedTodoIds.length)
+        ..color = BadgeColor.SECONDARY
+        ..overlap = BadgeOverlap.CIRCLE
+        ..anchorOrigin =
+            BadgeAnchorOrigin(horizontal: 'right', vertical: 'bottom'))(
+        props.children,
+      ),
+    ),
+  );
+}, $TaskCountBadgeConfig // ignore: undefined_identifier
+        ));
```

Comment:
🤔  This ended up with some odd formatting haha

Reply:
This is what the over_react formatter results are!

Reply:
I believe it, sometimes I've gotten some wonky stuff. @greglittlefield-wf do you see anything that would cause the formatter to do that? Might need a little patch to watch for the `uiFunction` usecases

Reply:
Would adding a comma after `$TaskCountBadgeConfig` and re-running over_react format fix this?

Reply:
oh yup, good call @sydneyjodon-wk !